htop.1.in

```diff
@@ -656,10 +656,12 @@ Memory sizes in
 .B htop
 are displayed in a human-readable form.
 Sizes are printed in powers of 1024. (e.g., 1023M = 1072693248 Bytes)
```

Comment:
If we are mentioning the default unit is "K as in KiB", then the "e.g." can be removed for brevity (as 1 KiB = 1024 bytes is noted right below)
```suggestion
Sizes are printed in powers of 1024.
```

Reply:
That at least resolves the long-standing nitpick, that the period should go after the parens … ;-) Anyway, what about mentioning IEC units?